src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/measure/ActionMeter.kt

```diff
@@ -140,67 +129,43 @@ class ActionMeter private constructor(
                 result = actionResult
             ).virtualUser(virtualUser)
             postMetricHook.run(actionMetricBuilder)
-            conditionallyAddDrilldown(actionMetricBuilder)
             output.write(actionMetricBuilder.build())
             throw Exception("Action '${key.label}' $actionResult", e)
         }
     }
 
-    private fun conditionallyAddDrilldown(actionMetricBuilder: ActionMetric.Builder): ActionMetric {
```

Comment:
When you delete more lines than you add, while maintaining functionality, you know you did something right 🙂